lib/perl/Genome/WorkflowBuilder/Converge.pm

```diff
@@ -73,4 +74,37 @@ sub notify_output_link {
     return;
 }
 
+sub get_ptero_builder_task {
+    require Ptero::Builder::Detail::Workflow::Task;
```

Comment:
We were trying to play nice with loading unneeded libraries.  Since you have to set an environment variable to use PTero currently, the Ptero::* modules don't need to be loaded until then.  I'm not sure the tiny savings is worth it though, what do you think?  It was always intended that these imports would be moved to to top of the module once PTero became the default.